Tracing: Add setting for sampling server (#29011)
grafana/grafana: docs/sources/administration/configuration.md

May be set with the environment variable `JAEGER_SAMPLER_PARAM`.

### sampling_server_url

> sampling_server_url is the URL of a sampling manager that providing a sampling strategy.

Reviewer: ```suggestion
sampling_server_url is the URL of a sampling manager providing a sampling strategy.
```

> sampling_server_url is the URL of sampling manager that can provide sampling strategy to this service.

Reviewer 2: ```suggestion
sampling_server_url is the URL of a sampling manager that providing a sampling strategy to this service.
```

Reviewer: ```suggestion
sampling_server_url is the URL of a sampling manager providing a sampling strategy to this service.
```

Reviewer: ~that~
